src/blockchain/examples/simple_node.rs

```diff
@@ -18,42 +18,58 @@ extern crate pretty_env_logger;
 extern crate pyrsia_blockchain_network;
 extern crate tokio;
 
+use std::error::Error;
+
 use futures::StreamExt;
 use libp2p::{
     core::upgrade,
     floodsub::{self, Floodsub},
     identity,
     mdns::Mdns,
-    mplex, noise,
-    swarm::{SwarmBuilder, SwarmEvent},
-    tcp::TokioTcpConfig,
-    Multiaddr, PeerId, Transport,
+    mplex, Multiaddr,
+    noise,
+    PeerId,
+    swarm::{SwarmBuilder, SwarmEvent}, tcp::TokioTcpConfig, Transport,
 };
-use std::error::Error;
 use tokio::io::{self, AsyncBufReadExt};
 
+use pyrsia_blockchain_network::network::Behaviour;
+
+use pyrsia_blockchain_network::block::{Block, PartialTransaction, Transaction, TransactionType, get_publickey_from_keypair};
+use pyrsia_blockchain_network::blockchain::Blockchain;
+use pyrsia_blockchain_network::blockchain::generate_ed25519;
 use pyrsia_blockchain_network::crypto::hash_algorithm::HashDigest;
-use pyrsia_blockchain_network::*;
 
-pub const CONTINUE_COMMIT: &str = "1"; // Allow to continuously commit
+pub const BLOCK_FILE_PATH: &str = "./blockchain_storage";
+pub const CONTINUE_COMMIT: &str = "1";
+// Allow to continuously commit
 pub const APART_ONE_COMMIT: &str = "2"; // Must be at least one ledger apart to commit
 
 #[tokio::main]
 async fn main() -> Result<(), Box<dyn Error>> {
```

Comment:
There is no return can you chop this out as well please

Reply:
also, you can add return OK(());